autodispose-rxlifecycle/src/main/java/com/ubercab/autodispose/rxlifecycle/RXLifecycleInterop.java

```diff
@@ -1,34 +1,34 @@
-package com.ubercab.autodispose.rxlifecycleinterop;
+package com.ubercab.autodispose.rxlifecycle;
 
 import com.trello.rxlifecycle2.LifecycleProvider;
 import com.trello.rxlifecycle2.OutsideLifecycleException;
-import com.trello.rxlifecycle2.RxLifecycle;
-import com.uber.autodispose.AutoDispose;
 import com.uber.autodispose.LifecycleEndedException;
 import com.uber.autodispose.ScopeProvider;
 import io.reactivex.Maybe;
 
 /**
- * Interop class for {@link RxLifecycle}. It provides static utility methods to convert {@link
- * LifecycleProvider} to {@link ScopeProvider}
- * <p> There are several static utility converter
- * methods such as {@link #bindLifecycle(LifecycleProvider)} for {@link
- * LifecycleProvider#bindToLifecycle()} and {@link #bindUntilEvent(LifecycleProvider, Object)} for
- * {@link LifecycleProvider#bindUntilEvent(Object)} </p>
+ * Interop class for RxLifecycle. It provides static utility methods to convert {@link
+ * LifecycleProvider} to {@link ScopeProvider}.
  *
- * <em>Unlike {@link AutoDispose}, {@link RxLifecycle} treats the {@link OutsideLifecycleException}
+ * <p>There are several static utility converter
+ * methods such as {@link #fromBindLifecycle(LifecycleProvider)} for {@link
+ * LifecycleProvider#bindToLifecycle()} and {@link #fromBindUntilEvent(LifecycleProvider, Object)} for
+ * {@link LifecycleProvider#bindUntilEvent(Object)}.
+ * </p>
```

Comment:
Remove this, add a `<p>` below

Reply:
Fixed - https://github.com/uber/AutoDispose/pull/118/commits/7c73cc226ba24b39b180d3904df96f1cdaf47824. Code review in github can be a pain.

Reply:
Much better than phabricator though! :)